recirq/qaoa/placement.py

```diff
@@ -106,25 +52,9 @@ def place_on_device(circuit: cirq.Circuit,
         initial_map: Initial placement of qubits
         final_map: The final placement of qubits after action of the circuit
     """
-    tk_circuit = pytket.extensions.cirq.cirq_to_tk(circuit)
```

Comment:
I suggest using a flag or two different functions here for tket and cirq, to support backwards compatibility.